features/admin.sms-templates.v1/pages/sms-customization.tsx

```diff
@@ -93,6 +95,9 @@ const SMSCustomizationPage: FunctionComponent<SMSCustomizationPageInterface> = (
     const isReadOnly: boolean = !smsFeatureConfig.enabled || !hasUpdatePermission;
     const hasSmsTemplateCreatePermissions: boolean = smsFeatureConfig.enabled && hasCreatePermission;
 
+    const organizationType: string = useSelector((state: AppState) => state?.organization?.organizationType);
+    const isSubOrgUser: boolean = (organizationType === OrganizationType.SUBORGANIZATION);
```

Comment:
Shall we use `useGetCurrentOrganizationType` hook instead?

```suggestion
	const { isSubOrganization } = useGetCurrentOrganizationType()
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7345/commits/842c3356a4eca454a317e9ef60fa2567980034de